tests/cli/commands/test_task_command.py

```diff
@@ -184,6 +184,20 @@ def test_task_state(self):
             'tasks', 'state', 'example_bash_operator', 'runme_0',
             DEFAULT_DATE.isoformat()]))
 
+    def test_task_states_for_dag_run(self):
+        with redirect_stdout(io.StringIO()) as stdout:
+            task_command.task_states_for_dag_run(self.parser.parse_args([
+                'tasks', 'states_for_dag_run', 'example_bash_operator', DEFAULT_DATE.isoformat()]))
+
+        # can't get perfect match as start/end date are variable
```

Comment:
I think you should save this data to the database in the code of test, and then the data will be constant.

Reply:
is there example?

Reply:
```
        dag = DagBag(load_examples=True).dags['example_bash_operator']
        dag.clear();
        dr = dag.create_dagrun(run_id='test_task_state_for_dagrun',
                               state=State.RUNNING,
                               execution_date=now,
                               start_date=now)

        ti_op1 = dr.get_task_instance(task_id=op1.task_id)
        ti_op1.set_state(state=State.REMOVED, session=session)
        ti_op2 = dr.get_task_instance(task_id=op2.task_id)
        ti_op2.set_state(state=State.SUCCESS, session=session)
```